packages/grid/x-data-grid-premium/src/components/GridAggregationColumnMenuItem.tsx

```diff
@@ -0,0 +1,91 @@
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import { unstable_useId as useId } from '@mui/material/utils';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { useGridApiContext } from '../hooks/utils/useGridApiContext';
+import { useGridRootProps } from '../hooks/utils/useGridRootProps';
+import {
+  getAvailableAggregationFunctions,
+  canColumnHaveAggregationFunction,
+} from '../hooks/features/aggregation/gridAggregationUtils';
+import { gridAggregationModelSelector } from '../hooks/features/aggregation/gridAggregationSelectors';
+import { GridAggregationModel } from '../hooks/features/aggregation/gridAggregationInterfaces';
+
+interface GridAggregationColumnMenuItemsProps {
+  column: GridColDef;
+  label: string;
+}
+
+export const GridAggregationColumnMenuItem = (props: GridAggregationColumnMenuItemsProps) => {
+  const { column, label } = props;
+  const apiRef = useGridApiContext();
+  const rootProps = useGridRootProps();
+  const id = useId();
+  const aggregationModel = useGridSelector(apiRef, gridAggregationModelSelector);
+
+  const availableAggregationFunctions = column
+    ? getAvailableAggregationFunctions({
+        aggregationFunctions: rootProps.aggregationFunctions,
+        column,
+      })
+    : [];
+
+  const cleanAggregationItem = React.useMemo(() => {
+    if (!column || !aggregationModel[column.field]) {
+      return '';
+    }
+
+    const aggregationFunctionName = aggregationModel[column.field];
+    if (
+      canColumnHaveAggregationFunction({
+        column,
+        aggregationFunctionName,
+        aggregationFunction: rootProps.aggregationFunctions[aggregationFunctionName],
+      })
+    ) {
+      return aggregationFunctionName;
+    }
+
+    return '';
+  }, [rootProps.aggregationFunctions, aggregationModel, column]);
+
+  const handleAggregationItemChange = (event: SelectChangeEvent<string | undefined>) => {
+    const newAggregationItem = event.target.value || undefined;
+    const currentModel = gridAggregationModelSelector(apiRef);
+    const { [column.field]: columnItem, ...otherColumnItems } = currentModel;
+    const newModel: GridAggregationModel =
+      newAggregationItem == null
+        ? otherColumnItems
+        : { ...otherColumnItems, [column.field]: newAggregationItem };
+
+    apiRef.current.setAggregationModel(newModel);
+    apiRef.current.hideColumnMenu();
+  };
+
+  return (
+    <MenuItem disableRipple>
+      <FormControl size="small" sx={{ width: 150 }}>
+        <InputLabel id={`${id}-label`}>{label}</InputLabel>
+        <Select
+          labelId={`${id}-label`}
+          id={`${id}-input`}
+          value={cleanAggregationItem}
+          label={label}
+          onChange={handleAggregationItemChange}
+          onBlur={(e) => e.stopPropagation()}
+          fullWidth
+        >
+          <MenuItem value="">...</MenuItem>
```

Comment:
Maybe use "None" instead?

Reply:
In filtering we use an empty string
@joserodolfofreitas is proposing a clear icon next to the select